src/CMakeLists.txt

```diff
@@ -75,6 +75,24 @@ set_target_properties(
 	SOVERSION "1.2"
 )
 
+find_program(
+  CLANG_PATH
```

Comment:
You are overwriting `CLANG_PATH` here. I am not sure, if CMake distinguishes between variables set in different directories.
To be on the safe side, please rename to e.g. `CLANG_FORMAT_PATH`.